src/docs/react-vis/basic.md

```diff
@@ -216,9 +219,23 @@ We could actually pass a yDomain prop in the YAxis and the VerticalBarSeries com
 
 To read more about axes in React-Vis, consult the [Axes docs](https://uber.github.io/react-vis/documentation/api-reference/axes).
 
-## 3. Fine tune our chart
+<ul class="insert takeaways">
+  <li>With the React-Vis component __XYPlot__ you can insert charts in your application.</li>
+  <li>And again, these charts work like other React components - with props and callbacks.</li>
+  <li>XYPlot can have various children like XAxis or BarSeries.</li>
+  <li>Series-type components need to have a _data_ prop.</li>
+</ul>
+
+<ul class="insert further-readings">
+  <li>[Visualization guidelines](https://uber-common.github.io/vis-tutorial/#documentation/visualization-guidelines/do-clear-simple-charts) - beyond technique, advice on how to build meaningful charts</li>
+  <li>[React-Vis](https://uber.github.io/react-vis/) and its documentation</li>
+</ul>
 
-**For the rest of the tutorial, you can now jump to the next section.**
+## Optional section
+
+Feel free to skip to [lesson 5](https://uber-common.github.io/vis-tutorial/#/documentation/5-interactions)
```

Comment:
To make it worth both on the raw markdown and on github pages, you can put the link as simply `interactions.md` and it should work.